src/__experimental__/components/input/input-presentation.component.js

```diff
@@ -64,7 +66,7 @@ class InputPresentation extends React.Component {
         className={ this.classNames() }
         ref={ this.container }
         onMouseDown={ this.handleMouseDown }
-        { ...props }
+        { ...filteredProps }
```

Comment:
Why would you want to remove these events? Shouldn't the developer be able to add them?